File: client-app/src/app/models/pvcTask.ts
export interface PVCTask {
  id: string;
  widthSize: number;
  lengthSize: number;
  printType: string;
  corners: string;
  image: string;
  moreInfo: string;
  formatType: string;
  taskType: string;
  startDate: Date | null;
  endDate: Date | null;
  isShipping: boolean;
  isPayed: boolean;
  isGotInvoice: boolean;
  taskStatus: string;
  price: number;
  count: number;
  customerId: string;
  cutomerName: string;
}

File: Domain/PVCTask.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class PVCTask:JobTask
    {
        public double WidthSize { get; set; }//רוחב
        public double LengthSize { get; set; }//אורך
        public string PrintType { get; set; }//מבריק מט
        public string Corners { get; set; }//פינות עגולות 
        public string Image { get; set; }//תמונה
        public string MoreInfo { get; set; }//מידע נוסף
    }
}